pyproject.toml

```diff
@@ -0,0 +1,54 @@
+[tool.poetry]
+name = "test-release-action"
+version = "1.4.7"
```

Comment:
Are the name and version correct? Should not be jobbergate-cli and the version match the one in the chabgelog?